### PR TITLE
IGVF-250-fixture-naming

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/tests/fixtures/schemas/treatment.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 
 @pytest.fixture
-def treatment_1(testapp):
+def treatment(testapp):
```
Comment (meenakshikagda) [inline src/igvfd/tests/fixtures/schemas/treatment.py:5]: How about we make it treatment_chemical to match treatment_protein?

Comment (meenakshikagda) [reply src/igvfd/tests/fixtures/schemas/treatment.py:5]: Not sure longer string but more description is better treatment_lactate_1_hr? @jenjou do you think this is acceptable?

Comment (jenjou) [reply src/igvfd/tests/fixtures/schemas/treatment.py:5]: I like `treatment_chemical`. I'm having trouble thinking of cases where we want to specifically test the treatment duration, so I think that might be too much detail to include in the fixture name.

Comment (meenakshikagda) [reply src/igvfd/tests/fixtures/schemas/treatment.py]: Okay

**Comment by meenakshikagda** [comment]
Looks good

**Review by meenakshikagda — APPROVED** [review]

**Pull request merged**